contents/blog/transparent-enterprise-pricing.md

```diff
@@ -0,0 +1,125 @@
+---
+date: 2022-06-01
+title: Why we adopted transparent enterprise pricing (and you should too)
```

Comment:
This is a great article, but the title maybe could be punchier/more opinionated?

'Transparent enterprise pricing was an obvious decision for us - why is it so rare?'
'Why we think companies hurt themselves when they hide their enterprise pricing'
'Stop hiding enterprise pricing behind a sales call'
'Why we believe 'contact sales for pricing' is a thing of the past'

Reply:
Hmm, I like your thinking. Just dreaming up other options...

- It's time to kill 'contact sales' for pricing
- It's time you ditched 'contact sales' for pricing
- It's time SaaS companies stop hiding their prices
- We hate secretive SaaS pricing models, so we fixed it
- Secretive SaaS pricing is bad for everyone. We found a better way